### PR TITLE
Config auth data for signup and login with AUTH_RECORD_KEYS in env

### PR DESCRIPTION
**Pull request by Steven-Chan**
connect #393

### CODE REVIEW

**Comment by Steven-Chan** [comment]
@rickmak @cheungpat updated

I have simplified the logic, down to only check key existence and index existence (literally two separate functions), newly created auth record keys column are string type, but accept any column type for existing column.

**Comment by Steven-Chan** [comment]
Although those two new functions are only called in main.go, but I want to have test case connected to database, so I still put them in pq package.

File: pkg/server/skydb/conn.go [hunk 1]
```diff
@@ -176,11 +176,13 @@ type Conn interface {
...
-	//
-	// Migration would be done in dev mode
-	EnsureAuthRecordKeysValid(authRecordKeys [][]string) error
+	// EnsureAuthRecordKeysExist check if authRecordKeys exist in
+	// user record schema
+	EnsureAuthRecordKeysExist(authRecordKeys [][]string) error
+
+	// EnsureAuthRecordKeysIndexesExist check if indexes of authRecordKeys
+	// exist in user record schema
+	EnsureAuthRecordKeysIndexesExist(authRecordKeys [][]string) error
```
Comment (rickmak) [inline pkg/server/skydb/conn.go]: `EnsureAuthRecordKeysIndexesExist` -> `EnsureAuthRecordKeysIndexesMatch`

We should mention we will delete indexes here.

**Comment by Steven-Chan** [comment]
@rickmak updated

**Pull request merged**